### PR TITLE
Hello Microverse Project 

### PR DESCRIPTION
**Pull request by Htetaungkyaw71**
This is my first pull request for my first repository in Microverse program. This is My Hello Microverse project and I have done following things in this.

Added  a markup file (HTML)
Added a stylesheet file (CSS)
Added a .gitignore file
Added text.md file in .gitignore file
Setup HTML, CSS linters
Added a descriptive Readme file
Added license for this project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test.md
```
Comment (bobb-Rob) [inline .gitignore]: - [ ]  [REQUIRED] You did well adding this text.md file name to gitignore. But according to this project requirements, you are suppose to add a text.md file with the text "This file should be ignored by git". Kindly implement this.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+node_modules
```
Comment (bobb-Rob) [inline .gitignore:2]: - [ ] [REQUIRED] This pull request can be improved by updating the title and description.
Pull Request should have a descriptive title telling what changes are inside.
Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the description -> Mention the things you did in brief detail.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
+</html>
```
Comment (bobb-Rob) [inline index.html:13]: - [ ] Nice work including README file. However, please remove any default text that has nothing to do with your project, this does not include the heading. You should actually add some text to each of the headings as it relates to your project.
See image below - Please edit the getting started section as it relates to your project. 
image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Htetaungkyaw71 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Htetaungkyaw71 — COMMENTED** [review]
I changes some text in test.md and remove some heading

File: .github/workflows/linters.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:4]: ### Your Summary

Screenshot from 2022-09-20 18-04-59

### Descriptive Summary Example :heavy_check_mark:
summary-descriptive

- [x] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Welcome to Microverse :tada: :tada: :bouquet: :bouquet:
# Required Changes :recycle:

almost-there


## Hey @Htetaungkyaw71 :wave:

## Highlights :1st_place_medal:
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following Some Requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @Htetaungkyaw71,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - No linter errors. ✔️
 - Professional PR message. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**